RocketChat/Rocket.Chat.ReactNative
# [NEW] Option to show mentions badge

## Proposed changes

Applying a new feature introduced by [this PR](https://github.com/RocketChat/Rocket.Chat/pull/25329) in the main repo.

## Issue(s)	


## How to test or reproduce

- Go to a room
- Then go to RoomActionsView
- You will see the Notification button, click it
- Tap the toggle 'Mark as Unread', should appear a new button below 'Show badge for mentions'
- Change the status and check in the RoomsListView if the badge's behavior is the same as web behavior

## Screenshots

https://user-images.githubusercontent.com/47038980/171664499-311a995e-9533-4d47-b232-a22d383b3ce8.mov

## Types of changes




## Checklist



## Further comments
